### PR TITLE
Update & translate for Russian language

### PR DESCRIPTION
**Pull request by ia**
* **Please check if the PR fulfills these requirements**
- [x] The changes have been tested locally
- [x] There are no breaking changes

* **What kind of change does this PR introduce?**
Update translation for Russian language.

* **What is the current behavior?**
Some of text lines are not translated in Russian.

* **What is the new behavior (if this is a feature change)?**
Translate & update text lines in Russian.

* **Other information**:
I made all model builds with RU locally successfully but I could verify on a real device only those translated lines which are valid for `TS80P`. Where the lines were too long, we tried to use common "shortenings".

Huge thanks for big help goes to ❤️ Iliana.

### CODE REVIEW

File: Translations/translation_RU.json [hunk 1]
```diff
@@ -7,16 +7,16 @@
             "message": "Калибровка\nзавершена!"
         },
         "ResetOKMessage": {
-            "message": "Сброс OK"
+            "message": "Сброс завершён"
```
Comment (discip) [inline Translations/translation_RU.json:10]: This message is too long to be displayed, since it is not scrolling.
Have you tested it on your device?
```suggestion
            "message": "Сброс OK"
```

Comment (discip) [reply Translations/translation_RU.json:10]: Does it work for you the way you changed it, or are you going to revert that?

Comment (ia) [reply Translations/translation_RU.json]: Ooops... you're right, we did miss it, sorry. Thanks a lot for pointing this out! Now it should fit.

Comment (ia) [reply Translations/translation_RU.json]: > Does it work for you the way you changed it, or are you going to revert that?

Okay, hold on... let me just double check it to make 100% sure...

Comment (discip) [reply Translations/translation_RU.json]: Klasna ti rasporidilsja! 👍🏻

**Review by discip — APPROVED** [review]

File: Translations/translation_RU.json (hunk 1 above)
Comment (ia) [reply Translations/translation_RU.json]: Danke schon for checking after me. Really appreciate that. It's just nice to see good healthy code review process in this project every time. :pray: 

However, just out of curiosity and for the future - did you notice that it won't fit just by looking through the diff? Or do you have some of your own local sanitizing scripts to check the translated lines for length? If it's the first, then nice catch. If it's the latter - it could be used as part of tests.

Because for me one of the most confusing things during testing translations at first was the fact that different lines - due to what must be displayed on the other side of OLED - limited by different number of chars per line. But it was easy to figure it out so it's not a big problem.  I think I saw something about that in the code related to settings GUI but I didn't pay much of attention to that back then. But I guess that was it.

Bottom line is _it seems_ not so hard technically to parse `translate.json` and check every line for _"does this line fit into amount of 'how much chars allowed by menu constructor / GUI context for this' ?"_. But in this case probably effort won't be worth the result much since translations are being made not so often (not even talking about adding new lines). So, just another one accidental rant...

Anyway, as I promised I double-checked the build right from _this_ PR `Actions` tab with my **latest push** (even before the merge, just for my own conscience ;) - it works & fits.

Comment (discip) [reply Translations/translation_RU.json]: Sorry for answering so late.

No I did neither notice just by locking through the diff nor do I have the skills to program automated sanitizing scripts.
I did test this in the past, since I intended to change that text as well. 😊

Since you are the one skilled enough to do so however, I dare to ask you to implement such a sanitizing script, based on the number of characters allowed in different situations.

I do think that this would be worth being done, since I noticed in at least on language that not every line is displayed correctly due to cut away characters and I am almost certain that there are a whole lot more out there.

Comment (ia) [reply Translations/translation_RU.json]: > Sorry for answering so late.

Please, don't worry at all. I clearly understand that we all are human beings with our lives so people dedicate its time as much as they can.

> No I did neither notice just by locking through the diff nor do I have the skills to program automated sanitizing scripts. I did test this in the past, since I intended to change that text as well. 😊

Aha, got it.

> Since you are the one skilled enough to do so however, I dare to ask you to implement such a sanitizing script, based on the number of characters allowed in different situations.
 
> I do think that this would be worth being done, since I noticed in at least on language that not every line is displayed correctly due to cut away characters and I am almost certain that there are a whole lot more out there.

Interesting suggestion! I can't promise anything, but I've added this task to my `ToDo` list for `IronOS` already so maybe at some point it will be implemented somehow. :crossed_fingers:
